search-app/docs/api.md

```diff
@@ -4670,7 +4670,7 @@ __Sample response__
 ```
 #### <a name="subscription-access-control"></a> Subscription Access Control
 
-Access to subscription is granted through the provider via the INGEST_MANAGEMENT_ACL and EMAIL_SUBSCRIPTION_MANAGEMENT
+Access to subscription is granted through the provider via the INGEST_MANAGEMENT_ACL and SUBSCRIPTION_MANAGEMENT
```

Comment:
Can you add description as to how the permission is granted through the two ACLs? I don't think anyone reading this document can follow without more explanation.

Reply:
Done.